Automattic/jetpack
# Forms: Add a line on top of the response list when the actions menu is sticky

## Proposed changes:

* Add a line on top of the response list when the actions menu is sticky

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Scroll the page and check if the line was added on top of the response list when the actions menu gets sticky
